test/.unibeautifyrc.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
@szeck87 `.unibeautifyrc` config file only works in the project root directory right now.

Reply:
For the CI you mean?

Reply:
Correct. You're right, it does work properly using the editor extensions though.

Consider this a Unibeautify CI bug (or feature request) and ignore my comment above 😄 .

Reply:
Yea this file was just meant to be used in the test cases, nothing else.